### ISSUE TITLE
What was the choice of loss function?

### ISSUE THREAD

**Issue by gordon-lim** [hiarindam/document-image-classification-TL-SG#8]
It was never specified in the paper.

**Comment by saikat-roy** [comment]
Since we don't have access to the original code and also as this was a long time ago, I can't be absolutely sure. But I am fairly certain it was just Cross Entropy.
